CONTRIBUTING.md

```diff
@@ -1,49 +1,68 @@
 # OSML Contributing Guidelines
 
 Thank you for your interest in contributing to our project! This document will guide you through the process of
-contributing to our repository using a release-based GitFlow workflow. Please read the guidelines carefully in order to
+contributing to our repository using a release-based GitFlow workflow. Please read the guidelines carefully to
 submit your pull requests effectively.
 
 ## Table of Contents
 
-- [Release-based GitFlow](#release-based-gitflow)
-- [Branches](#branches)
+- [Trunk-Based GitFlow](#trunk-based-gitflow)
 - [Linting](#linting)
-- [Use Cases](#use-cases)
-   - [Develop a new feature](#develop-a-new-feature)
-   - [Develop multiple features in parallel](#develop-multiple-features-in-parallel)
-   - [Create and deploy a release](#create-and-deploy-a-release)
-   - [Production hot fix](#production-hot-fix)
 - [Code Style](#code-style)
 - [Commit Messages](#commit-messages)
 - [Issue Tracking](#issue-tracking)
 
-## Release-based GitFlow
+## Trunk-Based GitFlow
 
-We follow a release-based GitFlow branching model to manage our codebase. It involves different types of branches to
-ensure proper version control and collaboration. The primary branches in this workflow are:
+![Trunk-Based Development](https://github.com/aws-solutions-library-samples/osml-imagery-toolkit/assets/4109909/0b3c03ae-4518-471e-9331-da850f0d2e22)
 
-note- `main`: This branch represents the latest production-ready state of the project code.
-- `dev`: This branch is used as the integration branch, combining all feature branches for a specific release.
-- `release/vX.Y.Z`: Branches in the format of `release/vX.Y.Z` are used to prepare a specific release.
-- `feature/*`: Feature branches are created for the development of new features or significant enhancements.
-- `hotfix/*`: Hotfix branches are created for bug fixes against production code.
+We follow a trunk-based development model to manage our codebase.
+This guide provides a step-by-step example of implementing a Trunk-Based Development (TBD) workflow for development.
+The primary branches developers create directly off of `main` in this workflow are:
 
+- `feat/*`: `feat` branches are created for the development of new features or significant enhancements.
+- `chore/*`: `chore` branches represent simple updates to the code that do not impact functionality.
+- `build/*`: `build` branches represent updates `.github` workflow actions.
 
-## Branches
+### 1. Creating Feature #1 (Developer #1)
```

Comment:
Would update title since have removed the `#2`

Reply:
Thank you! Updating

Reply:
Done